manifests/init.pp

```diff
@@ -132,6 +132,17 @@
 #
 # $server_ssl_crl::             Defines the Apache mod_ssl SSLCARevocationFile setting in Foreman vhost conf file.
 #
+# $keepalive::                  Enable KeepAlive setting of Apache?
+#                               type:boolean
+#
+# $maxkeepaliverequests::       MaxKeepAliveRequests setting of Apache
+#                               (Number of requests allowed on a persistent connection)
+#                               type:integer
+#
+# $keepalivetimeout::           KeepAliveTimeout setting of Apache
```

Comment:
Should these parameters contain underscores separating the words?  Other parameters do, e.g. `passenger_start_timeout`